docs/content/users/topics/cms_specific_help.md

```diff
@@ -1,26 +1,26 @@
+* If you want DDEV to know about the project type, but not create settings files, use `disable_settings_management: true`. If you want DDEV to use the CMS-specific nginx configuration, but don't want it to touch anything else, you can put `disable_settings_management: true` in your `.ddev/config.yaml` (or run `ddev config --disable-settings-management`) and DDEV won’t try to create any of the CMS-specific settings files.
+* There is also an environment variable `$IS_DDEV_PROJECT` that can be used to fence off DDEV-specific behavior. For example, with `$IS_DDEV_PROJECT` empty, the important parts of `settings.ddev.php` and `AdditionalSettings.php` (for TYPO3) are not executed. This means that DDEV’s `settings.ddev.php` won't be invoked if it somehow ends up in a production environment or in a non-DDEV local development environment.
+* The `.ddev/.gitignore` is created by `ddev start` because it gitignores itself. So the intention is that you should _not_ check in the `.ddev/.gitignore` and it will be created on start _if_ `disable_settings_management` is `false`. This helps teams to share `.ddev` folder checked in by Git, even if the `.ddev/.gitignore` changes with different versions.
 
 ## CMS-Specific Help and Techniques
 
 ### Drupal Specifics
 
-* **Settings Files**: By default, DDEV will create settings files for your project that make it "just work" out of the box. It creates a `sites/default/settings.ddev.php` and adds an include in `sites/default/settings.php` to bring that in. There are guards to prevent the `settings.ddev.php` from being active when the project is not running under DDEV, but it still should not be checked in and is gitignored.
-* **Twig Debugging**: With the default Drupal configuration, it's very difficult to debug Twig templates; you need to use `development.services.yml` instead of `services.yml`. Add this line in your `settings.php` or `settings.local.php`. See discussion at [drupal.org](https://www.drupal.org/forum/support/module-development-and-code-questions/2019-09-02/ddev-twig-debugging) and of course the Drupal documentation.
+* **Settings Files**: By default, DDEV will create settings files for your project that make it “just work” out of the box. It creates a `sites/default/settings.ddev.php` and adds an include in `sites/default/settings.php` to bring that in. There are guards to prevent the `settings.ddev.php` from being active when the project is not running under DDEV, but it still should not be checked in and is gitignored.
+* **Twig Debugging**: With the default Drupal configuration, it’s very difficult to debug Twig templates; you need to use `development.services.yml` instead of `services.yml`. Add this line in your `settings.php` or `settings.local.php`. See discussion at [drupal.org](https://www.drupal.org/forum/support/module-development-and-code-questions/2019-09-02/ddev-twig-debugging) and the Drupal documentation.
```

Comment:
Twig debugging should be mentioned in the xdebug section or possibly moved there. Had a user stumble over it yesterday.

Reply:
Added a reminder to my notes!